### PR TITLE
VBID-3401 html5 player improvements

### PR DESCRIPTION
**Pull request by natlownes**
- remove old project
- implement the requests -> play -> pop/expire flow as a stream
- config is setup in src/index.coffee
- add manifest.json for cortex
- `make package` to build a Cortex app

### CODE REVIEW

File: src/proof_of_play.coffee [hunk 1]
```diff
@@ -0,0 +1,44 @@
+inject        = require 'honk-di'
+{Transform}   = require('stream')
+{XMLHttpAjax} = require './ajax'
+
+
+class ProofOfPlay extends Transform
+  @scope: 'SINGLETON'
+
+  http:    inject XMLHttpAjax
```
Comment (markchadwick) [inline src/proof_of_play.coffee]: You should inject Ajax and provide XMLHttpAjax in your binder

**Comment by jawsthegame** [comment]
+1

File: src/ad_request.coffee [hunk 2]
```diff
@@ -0,0 +1,65 @@
+inject        = require 'honk-di'
+{XMLHttpAjax} = require './ajax'
+
+
+class AdRequest
+
+  config: inject 'config'
+  http:   inject XMLHttpAjax
```
Comment (markchadwick) [inline src/ad_request.coffee]: Inject Ajax

File: src/ad_request.coffee [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  navigator: inject 'navigator'
+
+  url: ->
+    "#{@config.scheme}://#{@config.host}#{@config.path}"
```
Comment (markchadwick) [inline src/ad_request.coffee]: Think it's worth just having the full ad request URL as part of the config? Due to (long story I half remember about PoPs), this used to be required, but shouldn't any longer.

File: src/ad_request.coffee [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      type:      'POST'
+      url:       @url()
+      dataType:  'json'
+      data:      JSON.stringify(@body())
+
+  body: ->
+    network_id:         @config.networkId
+    api_key:            @config.apiKey
+    device_id:          @config.deviceId
+    number_of_screens:  @config.numberOfScreens
```
Comment (markchadwick) [inline src/ad_request.coffee]: It's worth hardcoding this to 1. I believe the field is deprecated

File: src/ad_request.coffee (hunk 3 above)
Comment (natlownes) [inline src/ad_request.coffee]: yep, definitely going to do that

File: src/proof_of_play.coffee (hunk 1 above)
Comment (natlownes) [inline src/proof_of_play.coffee]: you just talking about

```coffeescript
@bind(Ajax).to(XMLHttpAjax)
```
right?

**Pull request closed without merging**